src/address.ts

```diff
@@ -407,8 +407,8 @@ export class Address {
   latitude(max: number = 90, min: number = -90, precision: number = 4): string {
     return this.faker.datatype
       .number({
-        max: max,
-        min: min,
+        max,
+        min,
```

Comment:
```suggestion
        min,
        max,
```

That way it feels more natural to me. But not really important.

Reply:
For me too. I kept the order but simplified the signature. I will apply our suggestion.